### ISSUE TITLE
Data ingestion on stdin

### ISSUE THREAD

**Issue by innogrey** [automixer/gobi#1]
Compared to similar projects based on goflow2, gobi looks interesting, however so far I've encountered at least one major issue.
Data ingestion seems to work when provided through goflow2 transport file (despite continuous  "proto: cannot parse invalid wire-format data \ unexpected EOF" warnings). I cannot make it work, however, with netflow data provided through pipe. Or have I understood something inherently wrong (unfortunately, documentation is scarce)? For the clarity sake, I'm not using any container system and what I'm trying to do is simple: goflow2 -format pb [...] -transport.file "" | gobi -f gobi.yml.

**Comment by automixer** [comment]
Hi, unfortunately I'm unable to reproduce the issue. The problem seems to be related to data ingestion by gobi. Continuous "proto: cannot parse invalid wire-format data \ unexpected EOF" warnings" should not appear.

My test env is as follows:
-- GoFlow2 v1.1.0 / Gobi v1.0.0
-- command line:
```
% ./goflow2 -transport.file.sep= -format=pb -format.protobuf.fixedlen=true | ./gobi -f gobi.yml
INFO[0000] Starting Gobi -- Flows Monitoring Tool v.1.0.0 
INFO[0000] 1 prometheus exporter(s) registered...       
INFO[0000] Starting GoFlow2                             
INFO[0000] Starting collection                           hostname= port=2055 scheme=netflow
INFO[0000] Starting collection                           hostname= port=6343 scheme=sflow
```
-- gobi.yml:
```
global:
  metricsaddr: :9310
  metricspath: /metrics

producer:
  input: stdin

promexporters:
  - metricsname: first
    minbps: 0
    minpps: 0
    flowlife: 10s
    maxscrapeint:  60s
    labelset: ["SamplerAddress"]
```
Could you please describe me your test environment?

**Comment by innogrey** [comment]
Thank you for your quick response. I should've probably attached all command line parameters to begin with. Anyway, thanks to the info you provided I managed to narrow down the problem to "-transport.file.sep=" on the goflow2 side, which I omitted completely (and which, in fact, was the only difference between the example you've provided and what I was doing, so finding the problem wasn't particularly troublesome). For some reason, I assumed that since pb is a binary format, the newline character doesn't apply at all and that goflow would ignore it. Apparently I was mistaken :) 

As a sidenote, it's interesting that with the use of transport file, gobi still produces metrics despite the warnings. I guess handling a pipeline data stream differs from reading from a file. Either way, adding "-transport.file.sep=" solves the warning as well.
